lms/djangoapps/student_account/views.py

```diff
@@ -124,6 +124,7 @@ def login_and_registration_form(request, initial_mode="login"):
             'login_form_desc': json.loads(form_descriptions['login']),
             'registration_form_desc': json.loads(form_descriptions['registration']),
             'password_reset_form_desc': json.loads(form_descriptions['password_reset']),
+            'create_account_option': settings.FEATURES.get('ALLOW_PUBLIC_ACCOUNT_CREATION')
```

Comment:
nit: Can we rename this and all relevant occurrences to `account_creation_allowed`?